app/components/pages/ContactUs/ContactELife.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { H1 } from '@pubsweet/ui'
+
+import Paragraph from '../../ui/atoms/Paragraph'
+import List from '../../ui/atoms/List'
+
+const ContactELife = props => (
+  <React.Fragment>
+    <H1>Contact eLife</H1>
+
+    <Paragraph>
+      You can use the &quot;Check Status&quot; link under Manuscript Tasks to
+      find information about your article and how long different parts of the
+      editorial and review process can take. You can also read about the
+      system&apos;s requirements regarding browsers, emails, and PDFs if you are
+      having difficulties. If you have any further questions about submitting,
+      submitted, or accepted articles, please contact our editorial or
+      production teams:
+    </Paragraph>
+
+    <List.Unordered>
+      <li>editorial@elifesciences.org</li>
+      <li>production@elifesciences.org</li>
+      <li>editorial@elifesciences.org</li>
+    </List.Unordered>
```

Comment:
The original page makes some (almost certainly futile) attempt at obfuscating the email addresses. I assume we're OK with no longer doing this?

Reply:
Ah. I assumed that this was unintentional & just rendering wrongly. @hdrury1 ?

Reply:
@jsms90 Let's go with what is currently live, I think that is a safer assumption - We can always 'fix' the attempt at obfuscation later :)

Reply:
The other thing to mention is that these pages are now generated client side which makes things mush harder for the bots (compared to just updating a regex). Also, spam detection is much better these days.